### ISSUE TITLE
Unable to serialize plaso.lib.event.EventTag object

### ISSUE THREAD

**Issue by elceef** [log2timeline/plaso#677]
**Plaso version:**

1.3.0 Release

**Operating system Plaso is running on:**

Ubuntu 15.10

**Installation method:**

sudo apt-get install plaso

**Description of problem:**

Serialization error when trying to export data to Timesketch 2015.12

**Debug output/tracebacks:**

```
$ sudo psort.py -d -o timesketch Suits.plaso_full 
[INFO] Data files will be loaded from /usr/share/plaso by default.
[INFO] Hostname: XPS-1530
[INFO] Index: fdf6d5e98ebb41eaa1de67bb10dcc237
[DEBUG] Converted retries value: False -> Retry(total=False, connect=None, read=None, redirect=0)
[INFO] Starting new HTTP connection (1): 127.0.0.1
[DEBUG] "HEAD /fdf6d5e98ebb41eaa1de67bb10dcc237 HTTP/1.1" 404 0
[DEBUG] Converted retries value: False -> Retry(total=False, connect=None, read=None, redirect=0)
[DEBUG] "PUT /fdf6d5e98ebb41eaa1de67bb10dcc237 HTTP/1.1" 200 21
No handlers could be found for logger "elasticsearch.trace"
[INFO] Adding events to Timesketch..
[ERROR] ({'parser': u'olecf/olecf_summary', u'datetime': '1996-12-22T17:13:01+00:00', 'tag': <plaso.lib.event.EventTag object at 0x7f78e3652f10>, u'message': u'Title: Solver - Skoroszyt z przyk\xc5\u201aadami Author: Microsoft Corporation Comments: Przyk\xc5\u201aadowe modele ilustruj\xc4\u2026ce zastosowania dodatku Microsoft Excel Solver. Last saved by: Microsoft Corporation Application: Microsoft Excel Security: 0', 'inode': 76998, 'display_name': u'VSS12:TSK:/Program Files/Microsoft Office/Office14/SAMPLES/SOLVSAMP.XLS;VSS13:TSK:/Program Files/Microsoft Office/Office14/SAMPLES/SOLVSAMP.XLS;VSS14:TSK:/Program Files/Microsoft Office/Office14/SAMPLES/SOLVSAMP.XLS;VSS15:TSK:/Program Files/Microsoft Office/Office14/SAMPLES/SOLVSAMP.XLS', 'uuid': u'2868de3e43cf4baaab22bc357172bf18', 'author': u'Microsoft Corporation', 'hostname': u'XPS-1530', 'comments': u'Przyk\xc5\u201aadowe modele ilustruj\xc4\u2026ce zastosowania dodatku Microsoft Excel Solver.', 'filename': u'/Program Files/Microsoft Office/Office14/SAMPLES/SOLVSAMP.XLS', 'application': u'Microsoft Excel', u'source_short': u'OLECF', 'username': u'-', 'store_number': 22, 'data_type': u'olecf:summary_info', 'timestamp': 851274781000000, 'store_index': 509, u'source_long': u'OLECF Summary Info', 'last_saved_by': u'Microsoft Corporation', 'name': u'Summary Information', 'timestamp_desc': u'Document Last Printed Time', 'title': u'Solver - Skoroszyt z przyk\xc5\u201aadami', 'security': 0}, TypeError("Unable to serialize <plaso.lib.event.EventTag object at 0x7f78e3652f10> (type: <class 'plaso.lib.event.EventTag'>)",))
> /usr/local/lib/python2.7/dist-packages/elasticsearch/serializer.py(50)dumps()
-> raise SerializationError(data, e)
```

**Source data:**

Has been analyzed with "tagging" plugin. Some details regarding the storage file below.

```
Collection information:
	parser_selection = win7
	recursive = False
	preferred_encoding = UTF-8
	os_detected = Windows
	workers = 0
	output_file = Suits.plaso_full
	method = imaged processed
	preprocess = True
	version = 1.3.0
	cmd_line = /usr/bin/log2timeline.py Suits.plaso_full /xmnt/2/GYI313337.dd
	debug = False
	runtime = multi process mode
	parsers = bencode, binary_cookies, chrome_cache, chrome_preferences, custom_destinations, esedb, filestat, firefox_cache, java_idx, lnk, mcafee_protection, msiecf, olecf, openxml, opera_global, opera_typed_history, pe, plist, prefetch, recycle_bin, skydrive_log, skydrive_log_error, sqlite, symantec_scanlog, winevtx, winfirewall, winjob, winreg
```

```
Collection information:
	cmd_line = /usr/bin/psort.py --analysis tagging -o null Suits.plaso_full
	plugins = [u'tagging']
	Action = Adding tags to storage.
	method = Running Analysis Plugins

Parser counter information:
	Counter: Total Tags = 4704
	Counter: Application Execution = 2772
	Counter: Logon = 1429
	Counter: Logoff = 183
	Counter: Application Installed = 96
	Counter: System Start = 68
	Counter: Shutdown = 68
	Counter: Shell Start = 60
	Counter: Time Change = 52
	Counter: Document Printed = 32
	Counter: Name Resolution Timeout = 8
	Counter: Application Updated = 4
```

**Comment by joachimmetz** [comment]
@elceef Can you confirm this is plaso version 1.3. If so I recommend switching version 1.4 and try if you can reproduce the issue in that version, so that we try to get this fixed in version 1.5

**Comment by joachimmetz** [comment]
Also is this maybe related to: https://github.com/log2timeline/plaso/issues/607

**Comment by berggren** [comment]
I can't reproduce this with the latest elastic module/Timesketch.
This should be fixed by #607. Please re-open if this is still an issue.
